tools/freeze_requirements.py

```diff
@@ -0,0 +1,75 @@
+import os
+from concurrent.futures import ThreadPoolExecutor
+from subprocess import run
+from typing import Tuple
+
+from tools.lib import gitroot
+from tools.lib import ts_print as print
+
+
+def worker(cmd: Tuple[str, ...]) -> Tuple[str, int]:
```

Comment:
I think this could be replaced with `subprocess.run` directly?  -- unless you think the printing is important (typical unix philosophy is "quiet unless necessary")

Reply:
I still want to print std{out},err, but I did forget to move printing out of the worker (thought about it some more, it would be nice in this simple case)